docs/docs/training-data-format.mdx

```diff
@@ -380,26 +380,26 @@ stories:
 
 Each step can be one of the following:
 
-  - A [user message](#user-messages), represented by **intent** and **entities**.
-  - An [or statement](#or-statement), which includes two or more user messages under it
-  - A bot [action](#actions)
-  - A [form](#forms)
-  - A [slot was set](#slots) event
-  - A [checkpoint](#checkpoints), which connects the story to another story
+  - A [user message](#user-messages), represented by **intent** and **entities** or **user**.
```

Comment:
```suggestion
  - A [user message](#user-messages), represented by a combination of **intent** and **entities**, or **user**.
```

Some way of clarifying which two go together, doesn't have to be phrased exactly like this